components/aws/sagemaker/batch_transform/src/batch_transform.py

```diff
@@ -62,7 +62,13 @@ def main(argv=None):
   logging.info('Submitting Batch Transformation request to SageMaker...')
   batch_job_name = _utils.create_transform_job(client, vars(args))
   logging.info('Batch Job request submitted. Waiting for completion...')
-  _utils.wait_for_transform_job(client, batch_job_name)
+
+  try:
+    _utils.wait_for_transform_job(client, batch_job_name)
+  except:
+    raise
```

Comment:
ya it does that

Reply:
If the `wait_for_transform_job` throws an exception because it encounters a 500 error then we may prematurely exit the component before it actually completed. This is probably a condition that should be handled within the waiter itself (discussion about switching to the proper waiter method), but maybe in the `except:` block we could print the stack trace of the exception just in case.

Reply:
it prints stack trace